includes/class-sensei-assets.php

```diff
@@ -258,4 +258,30 @@ public function disable_frontend_styles() {
 		add_filter( 'sensei_disable_styles', '__return_true' );
 	}
 
+	/**
+	 * Gets the contents of the icon file at assets/images/<name>.svg
+	 * for the given name. Or empty string if file not found.
+	 *
+	 * @param string $name The name of the icon file at "assets/images/<name>.svg".
+	 * @return string The icon markup.
+	 */
+	public function get_icon( string $name = '' ) {
```

Comment:
Yeah we can extend this later to collect and enqueue the used icons on the page.

Reply:
> Since the icons we're using here are pretty small too, we could use the same thing @yscik is using... But we can do it after that is merged. WDYT?

Of course. I left out the optimization part for now. I think the best time for optimization of this would be once it's used in sufficient enough places.